docs/maps/import-geospatial-data.asciidoc

```diff
@@ -58,6 +58,55 @@ On the {kib} home page, you can upload a file and import it into an {es} index w
 . Use the file chooser to select a GeoJSON file.
 . Click *Import file*.
 
+[discrete]
+=== Draw features in a map
+
+Upload features into {es} by drawing lines, polygons, circles, bounding boxes, and points in a map.
+
+To create a new index for drawing:
+
+. <<maps-create, Create a map>>.
+. Click *Add layer*.
+. Select *Create index*.
+. Set *Index name*.
+. Click *Create index*.
+
+To open an existing index for drawing:
+
+. <<maps-create, Create a map>>.
+
+. Click *Add layer*.
+
+. Select *Documents*.
+
+. Select the data view that points to your index. A <<data-views, data view>> can point to one or more indices. For feature editing, the data view must point to a single index.
+
+. Click *Add layer*.
+
+. Set *Scaling* to *Limit results to 10,000*.
```

Comment:
Hopefully we can remove this when #123409 is merged.

Reply:
correct, along with https://github.com/elastic/kibana/issues/124061, the goal is to remove all of these constraints an allow any vector layer with documents source to allow feature editing.